cli/run.go

```diff
@@ -321,3 +327,20 @@ func cleanup(ctx context.Context, client *http.Client, url, requestID string) er
 	}
 	return nil
 }
+
+func withoutEnv(env []string, prefixes ...string) []string {
+	result := make([]string, 0, len(env))
+	for _, v := range env {
+		ok := true
+		for _, prefix := range prefixes {
+			if strings.HasPrefix(v, prefix) {
+				ok = false
+				break
+			}
+		}
+		if ok {
+			result = append(result, v)
+		}
+	}
+	return result
+}
```

Comment:
How about expressing this with [`slices.DeleteFunc`](https://pkg.go.dev/slices#DeleteFunc)?

Reply:
See bdf673f7400b5fd5c5a3ce0f89f81cd849a3bea9